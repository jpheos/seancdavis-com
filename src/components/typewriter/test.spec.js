import React from "react"
import renderer from "react-test-renderer"

import Typewriter from "./"

describe("Typewriter", () => {
  it("renders correctly with no props", () => {
    const tree = renderer.create(<Typewriter />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
