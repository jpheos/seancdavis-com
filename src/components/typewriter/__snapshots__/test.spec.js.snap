// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typewriter renders correctly with no props 1`] = `
<span
  className="wrapper"
>
  <span
    className="inline-block"
  >
    
  </span>
  <span
    className="cursor isBlinking"
  />
</span>
`;
